_includes/code/components/validation.html

```diff
@@ -1,3 +1,6 @@
 {% capture validation %}
   {% library_component usa-validation %}
-{% endcapture %}{{ validation | strip }}
```

Comment:
Right now, in the validation component, we skip from an h2 ("Component preview") to an h4 ("Code requirements"). Can we make the h4 an h3? Or is there an intention for "Enter a code" to be an h3?

Reply:
Good catch. I updated the header in the component preview to be an `h3`. Noting that this does add the header to the in-page navigation link list. Is that ok?

Reply:
@amyleadem - similar to the summary box page, could we add an h3 that reads "Component example" above the preview, and then change "Code requirements" to an h4?

Reply:
@sarah-sch I've added "Default" headers to the component previews in Validation and Summary box, per our offline conversation in [c2bf69f](https://github.com/uswds/uswds-site/pull/2608/commits/c2bf69fe99300496d3fb025be8daa276d8a15532). Let me know if you need anything else!